src/introductory-offer-banner/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { Fill, Card, Button } from '@wordpress/components';
+import { __, sprintf } from '@wordpress/i18n';
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+import { useSelect, useDispatch } from '@wordpress/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+export const WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN =
+	'wc_calypso_bridge_introductory_offer_banner_hidden';
+
+export const CalypsoBridgeIntroductoryOfferBanner = () => {
+	const { updateOptions } = useDispatch( OPTIONS_STORE_NAME );
+	const { isModalHidden } = useSelect( ( select ) => {
+		const { getOption, hasFinishedResolution } =
+			select( OPTIONS_STORE_NAME );
+
+		return {
+			isModalHidden:
+				getOption(
+					WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN
+				) === 'yes' ||
+				! hasFinishedResolution( 'getOption', [
+					WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN,
+				] ),
+		};
+	} );
+
+	const dismissModal = () => {
+		updateOptions( {
+			[ WC_CALYPSO_BRIDGE_INTRODUCTORY_OFFER_BANNER_HIDDEN ]: 'yes',
+		} );
+	};
+
+	return (
+		! isModalHidden && (
+			<Fill name="woocommerce_homescreen_experimental_header_banner_item">
+				<Card>
+					<div className="wc-calypso-bridge-woocommerce-admin-introductory-offer-banner">
+						<div className="wc-calypso-bridge-woocommerce-admin-introductory-offer-banner__text">
```

Comment:
Font weight should be `600` here

Reply:
Updated in [Update font weight](https://github.com/Automattic/wc-calypso-bridge/pull/1397/commits/fdc305fb16417d0c030ae5de9cddb69dcafe38ab)